### PR TITLE
Chore: issue template update

### PR DESCRIPTION
**Pull request by Fatman13**
## Related Issues

resolve https://github.com/filecoin-project/venus/issues/5077

## Checklist

Before you mark the PR ready for review, please make sure that:
- [ ] All commits have a clear commit message.
- [ ] The PR title is in the form of of `<PR type>: <#issue number> <area>: <change being made>`
    - example: ` fix: #1234 mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_, _misc_, _perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _venus_, _venus-messager_, _venus-miner_, _venus-gateway_, _venus-auth_, _venus-market_, _venus-sealer_, _venus-wallet_, _venus-cluster_, _api_, _chain_, _state_, _vm_, _data transfer_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in [venus docs](https://venus.filecoin.io) or [Discussion Tutorials.](https://github.com/filecoin-project/venus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,99 +1,92 @@
...
         required: false
 - type: textarea
   id: version
   attributes:
-    label: Venus Version
+    label: 版本 / Version
     render: text
-    description: Enter the output of `./venus version` and `./venus-sealer -v` if applicable.
+    description: |
+      填写 `./venus version` 和/或 `./venus-messager --version` 的输出内容。
```
Comment (diwufeiwen) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]: 这里建议删掉，因为有 eg，一般人都明白，我们组件很多，这里只放两个不太合适

**Review by diwufeiwen — COMMENTED** [review]
commit比较多，合并前最好整理下

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (Fatman13) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: 不知道为啥之前的commit他也显示了。我好像是新开了个分支。

Comment (Fatman13) [reply .github/ISSUE_TEMPLATE/bug_report.yml:54]: 我把这个删掉。

**Review by simlecode — APPROVED** [review]

**Review by diwufeiwen — APPROVED** [review]
LGTM

**Pull request merged**